src/plus/gk/account/authenticationProvider.ts

```diff
@@ -75,9 +75,10 @@ export class AccountAuthenticationProvider implements AuthenticationProvider, Di
 		// Ensure that the scopes are sorted consistently (since we use them for matching and order doesn't matter)
 		scopes = scopes.sort();
```

Comment:
I think we need to remove the `signUp` scope from scopes before continuing, because we use the scopes as a key to look up existing sessions, and we don't want that difference to end up forcing a different session.

Reply:
Done in commit: [Pulls signUp flag out of scopes once used](https://github.com/gitkraken/vscode-gitlens/pull/3089/commits/757e1ba6f0c1955855876985fb7f0c397c9979e2)